frontend/src/ui/Modal.tsx

```diff
@@ -77,8 +78,7 @@ export const Modal = forwardRef<ModalHandle, PropsWithChildren<ModalProps>>(({
                 <div css={{
                     backgroundColor: "white",
                     borderRadius: 4,
-                    width: 400,
-                    maxWidth: "100%",
+                    minWidth: "clamp(300px, 90%, 400px)",
```

Comment:
`width: 90%; min-width: 300px` would make the modal always be `90%` of the viewport width when that is larger than 300px.
The `400px` upper bound is used for the "remove block" and "discard changes" modals on the "edit page content" page.
The `clamp(...)` here acts basically like a media query for viewport widths > `450px`. I think :D

(also not sure why this is marked as "outdated" here because I haven't changed anything yet)

Reply:
Oh yeah of course... but then, why not use `width` instead of `minWidth` here? Like, with the same `clamp` expression. That should have the same effect? And would be less confusing (at least to me :D)

Reply:
Hm but I also want it to be able to grow for the "remove page" button, though this is only necessery if we keep the cancel button. 
I woudn't really mind if the "Diese Aktion kann nicht rückgängig gemacht werden!" text is wrapped in two lines, but I think the buttons should be side by side on larger screen sizes.

Reply:
Ok, you know what: apparently I'm confused about CSS and it's apparently working as you intended, so let's just say it's ok like this :D